core/src/logger.rs

```diff
@@ -3,10 +3,9 @@
 //! gets emitted globaly from the container.
 
 use chrono::Local;
-use std::fmt;
 
 /// trait that defines the logging functionality that holochain_core requires
-pub trait Logger: fmt::Debug + Send {
+pub trait Logger: Send {
```

Comment:
Why would a logger implement `Debug`? I'm curious why this was there to begin with.